app.js

```diff
@@ -1,19 +1,24 @@
-const express = require('express')
-const {getApi , getTopics, getArticleId, getArticles}  = require('./controller')
-const app = express()
+const express = require("express");
+const {
+  getApi,
+  getTopics,
+  getArticleId,
+  getArticles,
+  getComments,
+} = require("./controller");
+const app = express();
 // middleware
-app.use(express.json())
-
+app.use(express.json());
 
 // api Endpoint
-app.get('/api', getApi)
-app.get('/api/topics', getTopics)
-app.get('/api/articles', getArticles)
-app.get('/api/articles/:article_id', getArticleId)
-
+app.get("/api", getApi);
+app.get("/api/topics", getTopics);
+app.get("/api/articles", getArticles);
+app.get("/api/articles/:article_id", getArticleId);
+app.get("/api/articles/:article_id/comments",getComments);
 
 app.use((req, res, next) => {
-    res.status(404).send({error: "Bad request!!! Not Found"} );
-next()
+  res.status(404).send({ error: "Bad request!!! Not Found", getComments });
```

Comment:
why are we sending the getcomments function here? Refer back to the error handling notes to set up this middleware more efficiently and follow good separation of concerns: https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro